### ISSUE TITLE
Instructions

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#570]
Issue group: Sending
Visibility: 2
Evaluation: Does not support
Success Criteria: 3.3.2: Labels or Instructions
Needs design: Interaction, Content

### Description
For uploading a list of recipients, none of the instructions are associated with the input, and they all follow the input in reading order. Consider placing the instructions and examples ahead of the input, to make it clearer what is needed to be uploaded. At least the file formats accepted, should be associated with the input using [aria-describedby](https://www.w3.org/TR/wai-aria-1.1/#aria-describedby). Additionally, the [accept attribute](https://html.spec.whatwg.org/multipage/input.html#attr-input-accept) should be specified on the file input. This will allow the file picker to filter the files the user can select for upload.

![](https://dl.airtable.com/.attachments/5d64d9ce98d6a52f3014f855bee95495/246190fa/Capturedecranle2022-03-15a13_55_03.png)

**Comment by amazingphilippe** [comment]
We can tackle this with #741
